engine/gameobject/src/gameobject/comp_script.cpp

```diff
@@ -246,11 +244,15 @@ namespace dmGameObject
 
     UpdateResult CompScriptUpdate(const ComponentsUpdateParams& params, ComponentsUpdateResult& update_result)
     {
+        CompScriptWorld* script_world = (CompScriptWorld*)params.m_World;
+        dmScript::UpdateScriptWorld(script_world->m_ScriptWorld, params.m_UpdateContext->m_DT);
```

Comment:
the timers are a script extension, and its "update" function needs to be called during the update flow